src/Enforcers/ModelValidationEnforcer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace PDGA\DataObjects\Enforcers;
+
+use \ReflectionClass;
+use \ReflectionAttribute;
+
+use PDGA\DataObjects\Attributes\Column;
+use PDGA\DataObjects\Enforcers\ValidationEnforcer;
+use PDGA\DataObjects\Validators\MaxLengthValidator;
+
+class ModelValidationEnforcer extends ValidationEnforcer
+{
+    /**
+     * Extends the ValidationEnforcer getValidationMetadata function,
+     * returning the array from the parent but adding a MaxLengthValidator
+     * if a property is defined as a Column and that column has a non-null
+     * value for maxLength.
+     *
+     * @param string $className - The name of the class for which metadata should be
+     * returned.
+     * @return array See ValidationEnforcer@getValidationMetadata.
+     */
+    public function getValidationMetadata(string $className): array
+    {
+        $metadata = parent::getValidationMetadata($className);
+
+        $ref   = new ReflectionClass($className);
+        $props = $ref->getProperties();

```

Comment:
Can't this come from the parent's metadata?  Each property (key) in `$metadata` now has a `reflectionProperty`.  That would avoid creating a second `ReflectionClass` and array of `ReflectionProperty`.

Likewise, the `foreach` loop below could iterate over the `$metadata` getting both the `$propName` and the `$meta` for that property (`validators` and such).

```
foreach($metadata as $propName => $propMeta)
```